tests/test_visitors/test_ast/test_keywords/test_raise.py

```diff
@@ -17,6 +18,14 @@ def check_exception_without_call():
     raise {0}
 """
 
+raise_exception_with_except = """
+def check_exception_without_call():
+    try:
+        x = 1
+    except Exception:
+        raise {0}
```

Comment:
NIP: it's not necessary to wrap `try except` in function definition:

```suggestion
raise_exception_with_except_raw = """
try:
    x = 1
except Exception:
    raise {0}
```